components/meter/meter-linear.js

```diff
@@ -115,6 +115,8 @@ class MeterLinear extends LitElement  {
 		// Floor so you don't get 100% when you are at 99.5%
 		context = context.replace('{%}', `${Math.floor(value / max * 100)}%`);
 		context = context.replace('{x/y}', `${value}/${max}`);
```

Comment:
Should we remove this?

Reply:
If no one else is using this currently, yeah.